core/eolearn/core/utils/testing.py

```diff
@@ -60,6 +63,9 @@ def generate_eopatch(
     # fill eopatch with random data
     # note: the patch generation functionality could be extended by generating extra random features
     for ftype, fname in parsed_features:
+        if ftype == FeatureType.META_INFO:
+            patch[(ftype, fname)] = "".join(random.choices(string.ascii_letters, k=20))
```

Comment:
use the `rng` that you have, or things are not repeatable.
Include a meta feature into the test where you ensure that seeds work, it should detect that